tests/test_event.py

```diff
@@ -17,55 +19,55 @@ def test_create_new_event(self):
 
     def test_create_from_json(self):
         now = time.gmtime()
-        data = json.dumps({ "@source": "test01.example.com",
-                 "@type" : "sometype",
-                 "@tags" : [ "tag1", "tag2", "tag3" ],
-                 "@fields" : { "key1" : "field1",
-                               "key2" : "field2",
-                               "key3" : "field3" 
-                             },
-                 "@timestamp" : time.strftime("%Y-%m-%dT%H:%M:%S",now),
-                 "@message" : "This is the message"
-                })
+        data = json.dumps({"@source": "test01.example.com",
+                           "@type": "sometype",
```

Comment:
The same thing applies to `@tags`, `@type`, and `@fields`, these `@` fields have been deprecated for a long time.